x/curating/client/cli/query.go

```diff
@@ -66,3 +64,70 @@ $ %s query curating params
 		},
 	}
 }
+
+// GetCmdQueryPost implements the post query command.
+func GetCmdQueryPost(storeName string, cdc *codec.Codec) *cobra.Command {
+	return &cobra.Command{
+		Use:   "post [vendor-id] [post-id]",
+		Args:  cobra.MinimumNArgs(2),
+		Short: "Query for a post by vendor ID and post ID",
+		Long: strings.TrimSpace(
+			fmt.Sprintf(`Query post by vendor ID and post ID.
+Example:
+$ %s query curating posts 1 123
+`,
+				version.ClientName,
+			),
+		),
+		RunE: func(cmd *cobra.Command, args []string) error {
+			cliCtx := context.NewCLIContext().WithCodec(cdc)
+
+			vendorID := args[0]
+			postID := args[1]
+
+			route := fmt.Sprintf("custom/%s/%s/%s/%s", storeName, types.QueryPost, vendorID, postID)
+			bz, _, err := cliCtx.QueryWithData(route, nil)
+			if err != nil {
+				return err
+			}
+
+			var post types.Post
+			cdc.MustUnmarshalJSON(bz, &post)
+			return cliCtx.PrintOutput(post)
+		},
+	}
+}
+
+// GetCmdQueryUpvote implements the upvotes query command.
+func GetCmdQueryUpvotes(storeName string, cdc *codec.Codec) *cobra.Command {
+	return &cobra.Command{
+		Use:   "upvote [vendor-id] [post-id]",
+		Args:  cobra.MinimumNArgs(1),
+		Short: "Query for an upvote by vendor ID and post ID",
```

Comment:
I think the description is different with what actually the command does which is returning a list of upvotes.